docs/docsite/rst/community/development_process.rst

```diff
@@ -225,6 +225,19 @@ While new modules, plugins, and roles are mentioned automatically in the generat
         # not have a period at the end.
         description: Wipes a server
 
+.. _contribution_guide_testing_prs:
+
+Testing PRs
+-----------
+
+Adding tests for your PR makes it a stronger candidate for being merged.
+
+PRs that add new functionality should include tests to fully exercise the new functionality and the failure paths of that functionality. Bugfix PRs should include tests that ensure the bug is no longer present. As a rule of thumb, integration tests should be preferred over unit tests.
```

Comment:
Long lines can be difficult to parse in diffs. Also, I think the docs repos are trying to follow a variant of semantic line breaks: https://docs.ansible.com/ansible-core/devel/dev_guide/style_guide/basic_rules.html#use-one-sentence-per-line.
```suggestion
PRs that add new functionality should include tests to fully exercise the new functionality and the failure paths of that functionality.
Bugfix PRs should include tests that ensure the bug is no longer present.
As a rule of thumb, integration tests should be preferred over unit tests.
```